README.md

```diff
@@ -31,3 +31,16 @@ To run operations from the command line, `./run local --help`.
 To run nose tests, `./run test`.
 
 To run pycodestyle, `./run lint`.
+
```

Comment:
add documentation on how to use ./run train

Reply:
That falls under ./run local --help - we didn't document model operations very well so far.

The trainer is also mostly intended to be used via HTTP by Uwazi - see test_e2e.py

Reply:
sorry, my fault.